### PR TITLE
Ft calibrate endpoint

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Adds calibrate path to staging and production ingresses

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):
- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Change the resource namespace from staging to development and apply the file

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

File: k8s/nginx-ingress/prod-frontend-ingress-resource.yaml [hunk 1]
```diff
@@ -8,12 +8,17 @@ spec:
...
-          - backend:
-              serviceName: airqo-platform-svc
-              servicePort: 80
-    - host: calibrate.airqo.net
-      http:
-        paths:
-          - backend:
-              serviceName: airqo-calibrate-app-svc
-              servicePort: 80
+          - path: /calibrate
```
Comment (123MwanjeMike) [inline k8s/nginx-ingress/prod-frontend-ingress-resource.yaml:11]: Mapping calibrate path on platform.airqo.net to calibrate service

File: k8s/nginx-ingress/prod-frontend-ingress-resource.yaml [hunk 2]
```diff
@@ -8,12 +8,17 @@ spec:
     - host: platform.airqo.net
       http:
         paths:
-          - backend:
-              serviceName: airqo-platform-svc
-              servicePort: 80
-    - host: calibrate.airqo.net
-      http:
-        paths:
```
Comment (123MwanjeMike) [inline k8s/nginx-ingress/prod-frontend-ingress-resource.yaml:16]: Remove calibrate subdomain approach

**Review by 123MwanjeMike — COMMENTED** [review]
Guiding comments

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM
Will we require some frontend changes in the Platform?

**Review by danielogen — APPROVED** [review]
LGTM

**Pull request merged**